docs/samples/vetkd-encrypted-notes.md

```diff
@@ -21,24 +19,17 @@ This example uses an [**insecure** implementation](https://github.com/dfinity/ex
 ## Prerequisites
 This example requires an installation of:
 
-- [x] Install the [IC SDK](https://internetcomputer.org/docs/current/developer-docs/setup/install/index.mdx).
+- [x] Install the [IC SDK](https://internetcomputer.org/docs/current/developer-docs/setup/install).
 - [x] Install `node.js` to build the web frontend.
 - [x] Download the following profile files from GitHub: https://github.com/dfinity/examples/
-- [x] If deploying in Rust, run `rustup target add wasm32-unknown-unknown` before getting started.
```

Comment:
why do you remove the Rust instructions? They work fine after running 
`export BUILD_ENV=rust`